sbws/lib/relaylist.py

```diff
@@ -25,17 +24,22 @@ def __init__(self, fp, cont, ns=None, desc=None):
         '''
         assert isinstance(fp, str)
         assert len(fp) == 40
-        assert stem_utils.is_controller_okay(cont)
         if ns is not None:
             assert isinstance(ns, RouterStatusEntryV3)
             self._ns = ns
         else:
-            self._ns = cont.get_network_status(fp, default=None)
+            try:
+                self._ns = cont.get_network_status(fp, default=None)
+            except Exception as e:
+                log.exception("Exception trying to get ns %s", e)
```

Comment:
If an exception is thrown, I'm not sure if `self` will end up with a `_ns` set to `None`, or if it will lack a `_ns` member. Please explicitly set `self._ns = None` if there was an exception.